src/components/Organisms/TodoCard/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+import AddTaskButton from "../../Atoms/AddTaskButton/index";
+import Task from "../../Molecules/Task/index";
+import COLOR from "../../../variables/color";
+
+const TodoCard = () => {
+  const [todoTask, setTodoTask] = useState([]);
+  const [taskAddOREdit, setAddOREdit] = useState(false);

```

Comment:
```suggestion
  const [taskAddOrEdit, setAddOrEdit] = useState(false);
```
厳密にはキャメルケースだとこうなると思います。